database/README.md

```diff
@@ -4,4 +4,4 @@
 Running `make dev` in the docker environment will create and persist the database.
 
 ## Database Diagram
-![](database_relationships.July2024.png)
+![](database_relationships.png)
```

Comment:
It would be nice to keep some information about the last time this was updated or changed. I think keeping it in the filename is fine but open to other ways of doing it as well.

Reply:
I'm working on a way to do this with a `make` command right now. When that's done (within a day or two), when it was last done should no longer be an issue.

Reply:
It's fixed with this PR: https://github.com/lucyparsons/OpenOversight/pull/1146